pages.hi/common/compare.md

```diff
@@ -0,0 +1,12 @@
+# compare
+
+> 2 छवियों के बीच अंतर देखें।
+> अधिक जानकारी: <https://imagemagick.org/script/compare.php>।
+
+- 2 छवियों की तुलना करें:
+
+`compare {{image1.png}} {{image2.png}} {{diff.png}}`
```

Comment:
Please translate the placeholders like `image` for get this easier for non-English speakers.

Reply:
Updated, thanks